llvm/bindings/ocaml/llvm/llvm.ml

```diff
@@ -296,6 +296,12 @@ module AtomicRMWBinOp = struct
   | UMin
   | FAdd
   | FSub
+  | FMax
+  | FMin
+  | UInc_Wrap
+  | UDec_Wrap
```

Comment:
Should the various FMax/FMin/UInc_Wrap/UDec_Wrap additions be a separate PATCH?

Reply:
I have created https://github.com/llvm/llvm-project/pull/105673 to add the previously missing bindings.